src/skill.h

```diff
@@ -127,20 +132,23 @@ class SkillLevel
         int _level = 0;
         int _exercise = 0;
         time_point _lastPracticed = calendar::turn;
-        bool _isTraining = true;
         int _knowledgeLevel = 0;
         int _knowledgeExperience = 0;
         int _rustAccumulator = 0;
 
+        // NOLINTNEXTLINE(cata-serialize)
+        bool _skillisTraining = true;
```

Comment:
Note: I renamed this variable just in case the cata-serialize lint does not also prevent *deserializing*.